guides/common/modules/proc_upgrading-rhel7-to-rhel8.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Upgrading-Project-from-RHEL7-to-RHEL8_{context}"]
+= Upgrading {Project} From {RHEL} 7 to {RHEL} 8
```

Comment:
Please ensure the title matches the anchor and file name closely. You can either use "project" in the file name or "server", whatever you feel is more descriptive.